src/Handler/Table/Create/CreateTableHandler.php

```diff
@@ -83,11 +85,18 @@ public function __invoke(
 
             // create table
             $db->executeStatement($createTableSql);
+
+            $ref = new TeradataTableReflection(
+                $db,
+                $databaseName,
+                $command->getTableName()
+            );
         } finally {
             if (isset($db)) {
                 $db->close();
             }
         }
-        return null;
+
+        return TableReflectionResponseTransformer::transformTableReflectionToResponse($databaseName, $ref);
```

Comment:
no, podle me by to melo vracet spis `ObjectInfoResponse` + `ObjectType::TABLE`, jako je to v https://github.com/keboola/php-storage-driver-teradata/blob/1f1ef3a51893de90e4f6859a40db25100d148ea8/src/Handler/Info/ObjectInfoHandler.php#L158-L158

Ted to vraci rovnou `TableInfo`, coz je matouci, pac to neni `*Response` a chova se to odlisne...

Reply:
Jo super diky :)